paramiko/server.py

```diff
@@ -301,9 +304,8 @@ def enable_auth_gssapi(self):
         :see: : `.ssh_gss`
         """
         UseGSSAPI = False
-        GSSAPICleanupCredentials = False
```

Comment:
If it's set in test and example but not a functional API member, should probably nuke from those latter as well.